README.md

```diff
@@ -344,146 +344,124 @@ to get started setting up your self-hosted runner.
-        docker-machine scp -r -q ~/.docker/machine/ $MACHINE:/docker_machine && \
-        docker-machine scp -q gce-credentials.json $MACHINE:/docker_machine/gce-credentials.json && \
-
-        eval "$(docker-machine env --shell sh $MACHINE)" && \
-        docker run --name runner -d \
-          -v /docker_machine/gce-credentials.json:/gce-credentials.json \
-          -e GOOGLE_APPLICATION_CREDENTIALS='/gce-credentials.json' \
-          -v /docker_machine/machine:/root/.docker/machine \
-          -e DOCKER_MACHINE=$MACHINE \
-          -e repo_token=$repo_token \
-          -e RUNNER_LABELS=$RUNNER_LABELS \
-          -e RUNNER_REPO=$RUNNER_REPO \
-          -e RUNNER_IDLE_TIMEOUT=120 \
-          dvcorg/cml-py3 && \
-        sleep 20 && echo "Deployed $MACHINE"
-        ) || (docker-machine rm -f $MACHINE && exit 1)
-```
 
-### Inputs
+### Arguments
+The function `cml-runner` accepts the following arguments:
+
+```
+Options:
+  --version            Show version number                             [boolean]
+  --labels             Comma delimited runner labels            [default: "cml"]
+  --idle-timeout       Time in seconds for the runner to be waiting for jobs
+                       before shutting down. 0 waits forever.     [default: 300]
+  --name               Name displayed in the repo once registered
+                                                     [default: "cml-cfwj9rrari"]
+  --driver             If not specify it infers it from the ENV.
+                                                   [choices: "github", "gitlab"]
+  --repo               Specifies the repo to be used. If not specified is
+                       extracted from the CI ENV.
+  --token              Personal access token to be used. If not specified in
+                       extracted from ENV.
+  --cloud              Cloud to deploy the runner      [choices: "aws", "azure"]
+  --cloud-region       Region where the instance is deployed. Choices:[us-east,
+                       us-west, eu-west, eu-north]. Also accepts native cloud
+                       regions.                             [default: "us-west"]
+  --cloud-type         Instance type. Choices: [m, l, xl]. Also supports native
+                       types like i.e. t2.micro
+  --cloud-gpu          GPU type.              [choices: "nogpu", "k80", "tesla"]
+  --cloud-hdd-size     HDD size in GB.
+  --cloud-ssh-private  Your private RSA SHH key. If not provided will be
+                       generated by the Terraform-provider-Iterative.
+                                                                   [default: ""]
+  --cloud-spot         Request a spot instance                         [boolean]
+  --cloud-spot-price   Spot max price. If not specified it takes current spot
+                       bidding pricing.                          [default: "-1"]
+  -h                   Show help                                       [boolean]
+  ```
+
+
+### Environmental variables
```

Comment:
Does this section also apply to the `Using CML with DVC` section above? Is any of this specific to self-hosted runners?

Reply:
It's actually the same! I'm not sure how to handle it- I don't like having duplicate docs. It's really that both CML + DVC and `cml-runner` use the same method of authenticating credentials to link to your cloud resources. Maybe I can just put a link... [Click here to see credentials required for supported cloud service providers].

Reply:
Yeah, I like the link idea.